pytest/tests/sanity/deploy_call_smart_contract.py

```diff
@@ -24,4 +24,30 @@
 hash_2 = base58.b58decode(hash_2.encode('utf8'))
 tx2 = sign_function_call_tx(nodes[0].signer_key, 'log_something', [], 100000000000, 100000000000, 20, hash_2)
 res = nodes[1].send_tx_and_wait(tx2, 10)
-assert res['result']['receipts_outcome'][0]['outcome']['logs'][0] == 'hello'
\ No newline at end of file
+assert res['result']['receipts_outcome'][0]['outcome']['logs'][0] == 'hello'
+
+compile_rust_contract('''
+extern "C" {
```

Comment:
Now that `near_bindgen` is used we can write:
```rust
use borsh::{BorshDeserialize, BorshSerialize};
#[near_bindgen]
#[derive(Default, BorshDeserialize, BorshSerialize)]
pub struct StatusMessage {}
#[near_bindgen]
impl StatusMessage {
  pub fn log_world(&self) {
    env::log(b"world");
  }
}
```

Reply:
Yes

Reply:
pytest uses bindgen contract now. i have a question here, i did not specify the contract name (status_message), why it knows i'm calling StatusMessage::log_world?

Reply:
Hmm I think i got it. this struct name StatusMessage really doesn't matter